src/net/sourceforge/kolmafia/textui/Parser.java

```diff
@@ -4934,7 +4929,7 @@ private void readToken() throws InterruptedException {
 
   /**
    * If we have an unread token saved in {@link #currentToken}, null the field, and delete it from
-   * its {@link Line#tokens}, effectively forgetting that we saw it.
+   * its {@link Line}, effectively forgetting that we saw it.
```

Comment:
This should probably stay. If we can't get it to ignore this type of error, I might consider removing the `@link` instead

Reply:
It generates a JavaDoc ERROR (!!!!!) that says something about protection and accessibility.  I couldn't figure it out in 5 minutes since JavaDoc is not a major part of my skill set.  I'd rather fix it than hide it.  Maybe @fredg1 (as author) has a suggestion?

Reply:
Semantically it doesn't make sense just saying "delete it from its Line" - its specifically deleting it from the line's list of tokens. I suppose we could say

```suggestion
its {@link Line#getTokensIterator()}, effectively forgetting that we saw it.
```

or

```suggestion
the tokens list on its {@link Line}, effectively forgetting that we saw it.
```